aioredis/commands/sorted_set.py

```diff
@@ -389,30 +352,17 @@ def zrevrangebylex(
         key,
         min=b"-",
         max=b"+",
-        include_min=True,
-        include_max=True,
         offset=None,
```

Comment:
Since we're breaking the API anyway, what do you think about using the same parameter names as redis-py?

https://github.com/andymccurdy/redis-py/blob/2c9f41f46991f94f0626598600d1023d4e12f0bc/redis/client.py#L2803-L2817

Reply:
Interesting question. It's not obvious what the right answer is, but with a bit of thought, my feeling is that we shouldn't change it, because SORT is in the same situation (`offset`/`count` in aioredis, `start`/`num` in redis-py). I don't really want to break SORT unless there is going to be an incompatible change anyway, and if we change Z*LEX but not SORT then aioredis isn't even internally consistent.

Incidentally the Redis command reference uses the terms "offset" and "count" (although that's just documentation, not protocol), so we'd just be trading consistency with one thing for consistency with another.